docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -0,0 +1,100 @@
+.. _supported_tasks:
+
+===================================
+Working with Labels and Predictions
+===================================
+
+Some checks, mainly the ones related to model evaluation, require labels and model predictions in order to run.
+In the deepchecks nlp package, predictions are passed into the suite / check ``run`` method as pre-computed
+predictions only (passing a fitted model is currently not supported).
+
+
+.. _supported_task__types:
+
+Supported Task Types
+====================
+
+Deepchecks currently supports two NLP task types:
+
+* **Text Classification**: Text classification is any NLP task in which a whole body of text (ranging from a sentence
+  to a document) is assigned a class (in the binary/multiclass case) or a certain set of classes (in the multilateral
+  case). In both the binary, the multiclass and the multilabel case the class "belongs" / "classifies" the whole text
+  sample. Examples for such tasks are Sentiment Analysis, Topic Extraction, detecting harmful content and more.
```

Comment:
```suggestion
  sample. Examples for such tasks are: Sentiment Analysis, Topic Extraction, harmful content detection and more.
```

Reply:
not sure if with or without capital, but the mix didn't look good IMO. feel free to reject...

Reply:
Think without capital because I made this up rather than this being an accepted term (as far as I know)